identity-service/src/config.js

```diff
@@ -821,6 +821,12 @@ const config = convict({
     format: Boolean,
     env: 'skipAbuseCheck',
     default: false
+  },
+  entityManagerReplicaSetEnabled: {
+    doc: 'Enable replica set updates with Entity Manager',
+    format: Boolean,
+    env: 'entityManagerReplicaSetEnabled',
+    default: false
```

Comment:
Thinking about it more, we should also disable user, playlist, track poa contract writes too... I guess they'd just fail on the contract themselves. 
There are old clients out there that will not have the code to write to the new entity manager flow yet. Maybe fine, but would ping client team about it